make/init.mk

```diff
@@ -0,0 +1,56 @@
+ROOT_DIR              := $(abspath $(dir $(lastword $(MAKEFILE_LIST)))/../)
+
+GO                    := GO111MODULE=on go
+
+CACHE_BASE            ?= $(ROOT_DIR)/.cache
+CACHE                 := $(CACHE_BASE)
+CACHE_BIN             := $(CACHE)/bin
+CACHE_INCLUDE         := $(CACHE)/include
+CACHE_VERSIONS        := $(CACHE)/versions
+CACHE_NODE_MODULES    := $(CACHE)/node_modules
+CACHE_NODE_BIN        := $(CACHE_NODE_MODULES)/.bin
+
+include $(ROOT_DIR)/.makerc
+
+# setup .cache bins first in paths to have precedence over already installed same tools for system wide use
+PATH                       := "$(PATH):$(CACHE_BIN):$(CACHE_NODE_BIN)"
+
+BUF_VERSION                ?= 0.35.1
+PROTOC_VERSION             ?= 3.13.0
+PROTOC_GEN_COSMOS_VERSION  ?= v0.3.1
+PROTOC_SWAGGER_GEN_VERSION ?= v2.2.0
```

Comment:
This version doesn't produce any useful output for me (it fails entirely, actually).

See: https://github.com/ovrclk/akash/pull/1175/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R29

Reply:
fixed